docs/source/howto/installation.rst

```diff
@@ -223,23 +223,269 @@ the configuration directory ``~/project_a/.aiida`` will be used.
+       For a given computer, you can increase how many seconds must pass between requests.
+       First load the computer in a shell with ``computer = load_computer(<COMPUTER_NAME>)``.
+       You can check the current value in seconds (by default, 10) with ``computer.get_minimum_job_poll_interval()``.
+       You can then set it to a higher value using::
+
+         computer.set_minimum_job_poll_interval(<NEW_VALUE_SECONDS>)
 
 .. _how-to:installation:update:
 
 Updating your installation
 ==========================
 
-`#4003`_
+1. Activate the Python environment where AiiDA is installed.
+2. Finish all running calculations.
+   All finished calculations will be automatically migrated, but it is not possible to resume unfinished calculations.
+3. Stop the daemon using ``verdi daemon stop``.
+4. :ref:`Create a backup of your database and repository<how-to:installation:backup>`.
+
+   .. warning::
+
+      Once you have migrated your database, you can no longer go back to an older version of ``aiida-core`` (unless you restore your database and repository from a backup).
+
+5. Update your ``aiida-core`` installation.
+
+    * If you have installed AiiDA through ``pip`` simply run: ``pip install --upgrade aiida-core``.
+    * If you have installed from the git repository using ``pip install -e .``, first delete all the ``.pyc`` files (``find . -name "*.pyc" -delete``) before updating your branch.
+
+6. Migrate your database with ``verdi -p <profile_name> database migrate``.
+   Depending on the size of your database and the number of migrations to perform, data migration can take time, so please be patient.
+
+After the database migration finishes, you will be able to continue working with your existing data.
+
+.. note::
+    If the update involved a change in the major version number of ``aiida-core``, expect :ref:`backwards incompatible changes<updating_backward_incompatible_changes>` and check whether you also need to update installed plugin packages.
+
+Updating from 0.x.* to 1.*
+--------------------------
+- <Additional instructions on how to migrate from 0.12.x versions `https://aiida.readthedocs.io/projects/aiida-core/en/v1.2.1/install/updating_installation.html#updating-from-0-12-to-1`>.
+- <Additional instructions on how to migrate from versions 0.4 -- 0.11 `https://aiida.readthedocs.io/projects/aiida-core/en/v1.2.1/install/updating_installation.html#older-versions`>.
+- For a list of breaking changes between the 0.x and the 1.x series of AiiDA, check <this page `https://aiida.readthedocs.io/projects/aiida-core/en/v1.2.1/install/updating_installation.html#breaking-changes-from-0-12-to-1`>.
 
 
 .. _how-to:installation:backup:
 
 Backing up your installation
 ============================
 
-`#4004`_
+A full backup of an AiiDA instance and AiiDA managed data requires a backup of:
+
+ * the profile configuration in the ``config.json`` file located in the ``.aiida`` folder.
```

Comment:
Check whether the bullet points are now showing (before they were somehow missing)

Reply:
It's ok now